### ISSUE TITLE
Embeddings differ when run in a batch or single image.

### ISSUE THREAD

**Issue by colojaro** [davidsandberg/facenet#278]
I am getting different results depending on how many images I test with in a batch or when I evaluate (order of execution). It almost seems as though batch normalization or something else is being updated as if in train mode. Has anyone had any luck running inference on a single image? Shouldn't the result be the same for a given image no matter when you run it or how many are in the batch?

**Comment by davidsandberg** [comment]
`phase_train` is passed using arg_scope in inception_resnet_v1. This is done on the line
```
        with slim.arg_scope([slim.batch_norm, slim.dropout],
                            is_training=is_training):
```
and it can easily be checked that is_training is set correctly by setting a break point in the batch_norm function.
I suggest that we close this issue and continue to track it in #234.
